dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```
@@ -70,7 +78,7 @@ FROM {{ .from }}
 
 ENV BEAT_SETUID_AS={{ .user }}
 
-{{- if contains .from "ubi-minimal" }}
+{{- if (or (contains .from "ubi-minimal") (contains .from "ubi9-minimal")) }}
```

I believe "ubi-minimal" is no longer relevant and if we go with `ubi9-minimal` when `ubi10-minimal` is released this won't be valid, how about the following
```suggestion
{{- if (and (contains .from "redhat/ubi") (contains .from "-minimal")) }}
```

---

good point